requirements-test.txt

```diff
@@ -26,6 +26,7 @@ web-fragments==0.3.1
 webob
 XBlock==1.2.9
 edx-opaque-keys==0.4.4
+edx-django-utils==3.13.0
```

Comment:
This is a requirement for `edx-rest-api-client` and its version is not pinned in `edx-rest-api-client` so it installs its latest version which is not compatible with python 3.5 that is why pinning it here.

Reply:
While this will upgrade it for the virtualenv, these are not installed with the block. So the block will still use the version in the platform which is 3.2.2 and the API client won't work for that. 

You will also need to upgrade the version in the platform by including it in setup.py, however that will upgrade this for the entire platform, and that might cause other breakages.

Reply:
Disregard the above, I think I messed something up, it's working properly.